src/scene/particle-system/particle-emitter.js

```diff
@@ -879,8 +879,9 @@ class ParticleEmitter {
                 customFace: this.emitter.orientation !== PARTICLEORIENTATION_SCREEN
             });
             this.shader = shader;
+            return shader;
```

Comment:
previous line can (should) be deleted?

Reply:
yep, I moved it outside of this function, as that's the consistent way to do it.